es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = number + street;
```

Comment:
@Jykwon24 You missed a couple concatenations:

---

![image](https://user-images.githubusercontent.com/16143771/156107403-ff95e372-b81b-4dd6-97c0-2f6eacd38f9f.png)